src/Forms/FormField.php

```diff
@@ -1439,7 +1439,13 @@ public function setSchemaData($schemaData = [])
     public function getSchemaData()
     {
```

Comment:
Is this method only ever called by formschema modal stuff? Feels like we should be doing this check in some more centralised place related explicitly to formschema modal logic.

Reply:
No it's not, it breaks elemental because it's called by `ElementalAreaField__holder.ss`

```
<div $AttributesHTML data-schema="$SchemaData.JSON">
    <%-- Field is rendered by React components --%>
</div>
```